packages/common/src/initiatives/defaults.ts

```diff
@@ -8,14 +8,15 @@ export const HUB_INITIATIVE_ITEM_TYPE = "Hub Initiative";
  * Default values of a IHubInitiative
  */
 export const DEFAULT_INITIATIVE: Partial<IHubInitiative> = {
-  name: "No title provided",
+  name: "",
   tags: [],
-  typeKeywords: ["Hub Initiative"],
+  typeKeywords: ["hubInitiativeV2"],
   catalog: { schemaVersion: 0 },
   permissions: [],
   schemaVersion: 2,
   status: HubEntityStatus.notStarted,
   features: InitiativeDefaultFeatures,
+  hero: HubEntityHero.map,
```

Comment:
Since the hero property is part of `IWithViewSettings`, I think this default should be within the `entity.view` object with `featuredContentIds`. Same with the model default below - it should be within `data.view`